source/modules/MEI.cmn.xml

```diff
@@ -1140,6 +1116,22 @@
       </attDef>
     </attList>
   </classSpec>
+  <classSpec ident="att.tremForm" module="MEI.cmn" type="atts">
```

Comment:
Why not `att.tremolo.logical`? `att.tremForm` looks weird somehow.

Reply:
While I agree in principle with @rettinghaus, we actually don't have any `att.tremolo.*` classes, and the only other one is `att.tremMeasured`. So I would say this change is at least consistent with what we currently have for tremolo, even if it isn't completely consistent with the rest of the spec.

Reply:
To be honest, I hadn't invested a whole lot of time into considering the best name for this. With a closer look now, I come to the conclusion that there is actually way more to do, and thus to discuss, than just this name. The (proposed) `att.tremForm` is decribed as _Attributes describing the form of a tremolo_. Then we have an attribute name _form_, which is described as _describes the style of the tremolo_. Putting aside the slight inconsistencies, that seems to indicate a _visual_ statement. The purpose, however, is a _logical_ one. So, the situation seems to require much more consideration than just the name of the att class, and I don't think we should do that _now_ – we clearly have many other situations like this, which we would want to fix as well, as soon as we run into them.

Reply:
Visual and logical can be the same… I don’t think we need to complicate this. Can we proceed with this change and open a new issue for the additional post-5 changes?

Reply:
I think this is ok for now as it has a local consistency, similar to what was acceptable for us in #1173 and resulting in #1194 for MEI6